packages/react-native-session-replay/src/specs/NativeDdSessionReplay.ts

```diff
@@ -45,5 +45,5 @@ export interface Spec extends TurboModule {
     stopRecording(): Promise<void>;
 }
 
-// eslint-disable-next-line func-names
+// eslint-disable-next-line import/no-default-export
```

Comment:
func-names was not needed anymore. I also decided to keep the default export for now.